packages/yourloops/models/message.ts

```diff
@@ -41,11 +41,4 @@ interface MessageNote {
   }
 }
 
-/**
- * An array of messages
- */
-interface MessagesThread {
-  messages?: MessageNote[];
-}
-
-export { MessageNote, MessagesThread };
+export { MessageNote };
```

Comment:
MessageNote structure do not seems to reflect well the go one:
See https://github.com/mdblp/message-in-a-bottle/blob/cbaad1b30094812cd47b7b840021ad8abccfdc87/schema/message.go#L6

```suggestion
export default MessageNote;
```